semantic-php/src/Language/PHP/Tags.hs

```diff
@@ -5,13 +5,15 @@
 {-# LANGUAGE OverloadedLists #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
Better 🔥 this; I think you’ve fixed most of the patterns here, if not all of them.